src/routes/users/readUser.ts

```diff
@@ -0,0 +1,91 @@
+import { FastifyInstance, FastifyReply, FastifyRequest } from 'fastify';
+import { Type, Static } from '@sinclair/typebox';
+import { knexController } from '../../database/database';
+
+import userTypes, { User, UserResponse } from '../../types/user.types';
+const { userResponse } = userTypes;
+
+const searchParamsPayload = Type.Object({
+  userId: Type.String(),
+});
+type SearchParamsPayload = Static<typeof searchParamsPayload>;
+
+const fetchAllResponse = Type.Array(userResponse);
+
+const searchSchema = {
+  description: 'Search for an user with given id',
```

Comment:
Are we searching for user though? Is this searching? 

I would rather use "Get user with the given id"